README.md

```diff
@@ -388,6 +390,14 @@ mvn dependency:copy-dependencies
     key.converter.avroRecordType=GENERIC_RECORD
     value.converter.avroRecordType=GENERIC_RECORD
 ```
+
+Kafka Connect schema registry convertes are typically configured with a URL to the desired schema registery endpoint. GSR does not have the notion of a URL endpoint. The service is instead accessed with AWS API calls specifying the name of the GSR resource. You can optionally specify the name of the desired GSR resource. If registry name isn't provided a default GSR resource is used.
```

Comment:
We can shorten this a bit,

"As Glue Schema Registry is a fully managed service by AWS, there is no notion of schema registry URLs. Name of the registry (within the same AWS account) can be optionally configured using following options. If not specified, `default-registry` is used."

Reply:
That is concise and gets the point across. Thank you for the suggestion, I've updated my PR with the change.